tutorials/networking/websocket.rst

```diff
@@ -37,7 +37,7 @@ This example will show you how to create a WebSocket connection to a remote serv
     extends Node
 
     # The URL we will connect to
-    export var websocket_url = "ws://echo.websocket.org"
+    export var websocket_url = "ws://www.piesocket.com/websocket-tester"
```

Comment:
Did you try if it works? The websocket connection URL is supposed to be:
`wss://demo.piesocket.com/v3/channel_1`, plus, it seems it requires an API key?
I think we could alternatively try: https://libwebsockets.org/testserver/ which is a server implementation using the previous library we were using (`libwebsockets`) (again, we should double check the connect URL).

Reply:
Ah yes, sorry. I'd forgotten that I copied the address from [https://www.piesocket.com/websocket-tester](https://www.piesocket.com/websocket-tester), so the functioning websocket url for this service would be

`wss://demo.piesocket.com/v3/channel_1?api_key=oCdCMcMPQpbvNjUIzqtvF1d2X2okWpDQj4AwARJuAgtjhzKxVEjQU6IdCjwm&notify_self`

The libwebsockets alternative looks a lot cleaner, but I haven't been able to find a URL for it; what did you use?